app/models/stats.rb

```diff
@@ -12,7 +12,9 @@ def initialize
   private
 
   def calculate_new_reg
-    25
+    from = 7.days.ago.beginning_of_day
+    to = 1.day.ago.beginning_of_day
```

Comment:
[NIT] An empty line here would help readability :) Will divide the assignment of variables from the actual execution :) 
```suggestion
    to = 1.day.ago.beginning_of_day
    
```